.github/workflows/ci.yaml

```diff
@@ -22,16 +22,6 @@ jobs:
         with:
           job-id: jdk11
 
-      - name: Build
-        uses: nick-invision/retry@v2.4.0
-        env:
-          S3_BUILD_CACHE_ACCESS_KEY_ID: ${{ secrets.S3_BUILD_CACHE_ACCESS_KEY_ID }}
-          S3_BUILD_CACHE_SECRET_KEY: ${{ secrets.S3_BUILD_CACHE_SECRET_KEY }}
-        with:
-          command: ./gradlew build --stacktrace
```

Comment:
Wait, you are removing the whole build step here :)

Reply:
well, that would fix the build, right? 🤦‍♂️